official/selection_to_pdf.lua

```diff
@@ -94,6 +101,14 @@ dt.register_storage("export_pdf","Export thumbnails to pdf",
       if my_title == "" then
         my_title = "Title"
       end
+      local thumbs_per_line = no_of_thumbs_widget.text
+      if thumbs_per_line == "" then
+          thumbs_per_line = 4
+      else
+          thumbs_per_line = tonumber(thumbs_per_line)
+      end
+      width = (1/thumbs_per_line) - 0.01
```

Comment:
Or use a slider with hard and soft boundaries? A user can put every string in the entry field. A slider gives back a value.